sources/web/datalab/polymer/modules/gapi-manager/gapi-manager.ts

```diff
@@ -569,12 +728,12 @@ class GapiManager {
     private static _load(): Promise<void> {
       return GapiManager.auth.loadGapiAndScopes('cloudresourcemanager', 'v1', GapiScopes.CLOUD);
     }
-
   };
 
   /**
    * The auth class is responsible for initializing authorization, including
    * loading the gapi code and any data needed for oauth.
    */
   public static auth = new ClientAuth();
+  // public static auth = new ServerAuth();
```

Comment:
Remove?

Reply:
I added this comment to make it clear what needs to be done to switch from client-side auth flow to server-side auth flow. I could change it to a prose comment instead if that would be better.

Reply:
I'll approve since I don't have a strong opinion on this.